doc/classes/TextEdit.xml

```diff
@@ -1222,6 +1234,20 @@
 				Provide custom tooltip text. The callback method must take the following args: [code]hovered_word: String[/code].
 			</description>
 		</method>
+		<method name="set_use_syntax_highlighter_font_change">
```

Comment:
Let's expose this as a editor property. Right now it can only be changed with code